cpp/include/cudf/detail/aggregation/aggregation.hpp

```diff
@@ -320,7 +320,7 @@ class m2_aggregation : public aggregation {
 /**
  * @brief Derived class for specifying a standard deviation/variance aggregation
  */
-class std_var_aggregation : public aggregation {
+class std_var_aggregation : public rolling_aggregation {
```

Comment:
Looks like there may be a conflict with the changes in PR #8906 
https://github.com/rapidsai/cudf/pull/8906/files#diff-327237da3867b61ac706975a6778112465cf0c46fce2886c5c0bfcb251937eb3L323
Perhaps 8906 should be merged first?

Reply:
Sure. That PR can merge first. I think it's trivial to merge the changes there to this one as there are other aggregations in that PR (argmin e.g.) that are subtyped after both `rolling_aggregation` and `groupby_aggregation`.